CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 
 ## master
 
+* Adds support for Code Insights for Bitbucket Server. [@qbeorama](https://github.com/Qbeorama)
```

Comment:
Great PR, and a really cool idea - any chance you can fill this CHANGELOG entry with user-docs on how to use it?

Reply:
Sure, added info on extra environmental variables need.